integration-tests/build.gradle

```diff
@@ -18,9 +18,5 @@ dependencies {
             "org.testcontainers:junit-jupiter",
             "com.squareup.okhttp3:okhttp"
 
-    tasks.withType(Test) {
```

Comment:
Can you remove the shadow plugin up top too (which I assume isn't needed anymore)?

Reply:
oh yes, definitely. good catch.

Reply:
done